frontend/app/forms/inputs/text-area.component.js

```diff
@@ -21,7 +21,7 @@ export default function TextArea(props) {
 
   React.useEffect(() => {
     setValue(get(dataContext.data, props.dataKey, ""));
-  }, [dataContext]);
+  }, [dataContext, props.dataKey]);
```

Comment:
Wow yes all these effect dependency array errors is a great reason why we should have had linting all along